egs/swbd/run.sh

```diff
@@ -10,6 +10,15 @@ num_word=20000
 lm_dir="data/lm/"
 arpa_dir="data/arpa/"
 
+max_memory='--max-memory=10G'
+# If you do not want to set memory limitation for "sort", you can use
+#max_memory=
+# Choices for the max-memory can be:
+# 1) integer + 'K', 'M', 'G', ... 
+# 2) integer + 'b', meaning using default unit byte and no multiplication
+# 3) integer + '%', meaning a percentage of memory
+# 4) integer, default unit is byte
```

Comment:
Make sure that it's not just a question of the comments being wrong... I
assumed that the default units were bytes... I hope you didn't convert to
bytes when dividing by the num-jobs and forget to add the 'b' in the option.
Dan


On Sun, Aug 14, 2016 at 7:44 AM, Wang Jian <notifications@github.com> wrote:

> In egs/swbd/run.sh
> <https://github.com/danpovey/pocolm/pull/46#discussion_r74703004>:
>
> > @@ -10,6 +10,15 @@ num_word=20000
> >  lm_dir="data/lm/"
> >  arpa_dir="data/arpa/"
> >
> > +max_memory='--max-memory=10G'
> > +# If you do not want to set memory limitation for "sort", you can use
> > +#max_memory=
> > +# Choices for the max-memory can be:
> > +# 1) integer + 'K', 'M', 'G', ...
> > +# 2) integer + 'b', meaning using default unit byte and no multiplication
> > +# 3) integer + '%', meaning a percentage of memory
> > +# 4) integer, default unit is byte
>
> Hi @keli78 <https://github.com/keli78>, I read manual page of sort(1)
> just now, it says the default unit of --buffer-size= is 'K' not 'b'. Please
> correct these lines of comment.
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/danpovey/pocolm/pull/46/files/aa4fdf234ded24c172f6292c43e76dad480e8915#r74703004>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ADJVu-5hIwoMLhQiVhDqqZv8Isgm-c0cks5qfynWgaJpZM4JibxT>
> .
>